cloud/aws/templates/aws_oidc/app.tf

```diff
@@ -84,7 +84,7 @@ module "civiform_server_container_def" {
     STORAGE_SERVICE_NAME = "s3"
     AWS_S3_BUCKET_NAME   = aws_s3_bucket.civiform_files_s3.id
 
-    CIVIFORM_IMAGE_TAG                      = var.image_tag
+    CIVIFORM_VERSION                        = var.image_tag
```

Comment:
Don't we still need `CIVIFORM_IMAGE_TAG` for use [here](https://github.com/civiform/civiform/blob/b556a3343e0c62a6f928f8750d8c1dd50a80919b/server/conf/application.conf#L485)? Or am I misunderstanding how it's hooked up?

Reply:
The [bin/build-prod](https://github.com/civiform/civiform/blob/main/bin/build-prod) script builds the civiform/civiform docker image we push to dockerhub.  It uses [prod.Dockerfile](https://github.com/civiform/civiform/blob/main/prod.Dockerfile#L46) to build the image.

In prod.Dockerfile, we have `ENV CIVIFORM_IMAGE_TAG=$image_tag` which sets the CIVIFORM_IMAGE_TAG environment variable inside of the docker container.  I previously added the `CIVIFORM_IMAGE_TAG = var.image_tag` line so that we can set the version to a nice version string like 'v1.18.0'.  This change adds a new CIVIFORM_VERSION to pass the version string instead of overwriting the image tag variable